lib/i18n/js.rb

```diff
@@ -188,6 +191,13 @@ def self.extract_segment_options(options)
       segment_options.merge(options.slice(*Segment::OPTIONS))
     end
 
+    def self.available_locales
+      config.fetch(:available_locales) do
+        # default value
+        I18n.available_locales rescue []
```

Comment:
If we get a `MethodMissing` (`I18n.available_locales`), we default to an empty array.